public/js/main.js

```diff
@@ -7,24 +7,27 @@ let query;
 let isShowingLongValues;
 let pluginConfiguration;
 const headersToAlignRight = [];
-const loaderElement = document.getElementById('loader');
-const modal = document.getElementById('modal');
-
-// close the edit columns modal when clicking outside of it
-document.querySelector('body').addEventListener('click', () => {
-    if (event.target === modal) {
-        closeModal();
-    }
-});
-// close the edit columns modal when escape key is pressed
-window.onkeyup = (e) => {
-    if (
-        e.key === 'Escape' && modal.style.visibility === 'visible'
-    ) {
-        e.preventDefault();
-        closeModal();
-    }
-};
+const g = typeof globalThis === 'object' ? globalThis : typeof window === 'object' ? window : null;
```

Comment:
we use either "globalThis" (nodejs and modern browsers) or "window", to be able to load this file in a browser or node